doc(install): add Longhorn CLI in quick installation section (#958)
longhorn/website: content/docs/1.7.0/v2-data-engine/quick-start.md

### Check Environment

> #### With The Longhorn Command Line Tool

Reviewer: ```suggestion
#### Using the Longhorn Command Line Tool
```

Author: Modified. Thanks.

> The `longhornctl` tool is a CLI interface to Longhorn operations. For more information, see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).

Reviewer 2: Likewise.
```suggestion
The `longhornctl` tool is a CLI for Longhorn operations. For more information, see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).
```

> The `longhornctl` tool is a CLI interface to Longhorn operations. For more information, please see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).
> 
> To check the prerequisites and configurations, please download the command line tool `longhornctl` and utilize the `check` sub-command:

Reviewer: ```suggestion
The `longhornctl` tool is a CLI interface to Longhorn operations. For more information, see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).

To check the prerequisites and configurations, download the tool and run the `check` sub-command:
```

Author: Modified. Thanks.

```shell
# For AMD64 platform
curl -sSfL -o longhornctl https://github.com/longhorn/cli/releases/download/v{{< current-version >}}/longhornctl-linux-amd64
# For ARM platform
curl -sSfL -o longhornctl https://github.com/longhorn/cli/releases/download/v{{< current-version >}}/longhornctl-linux-arm64

chmod +x longhornctl
./longhornctl check preflight --enable-spdk
```

> Example result:

Reviewer: ```suggestion
Example of result:
```

Author: Modified. Thanks.

```shell
INFO[2024-01-10T00:00:01Z] Initializing preflight checker
INFO[2024-01-01T00:00:01Z] Cleaning up preflight checker
INFO[2024-01-01T00:00:01Z] Running preflight checker
INFO[2024-01-01T00:00:02Z] Retrieved preflight checker result:
worker1:
  error:
  - 'HugePages is insufficient. Required 2MiB HugePages: 1024 pages, Total 2MiB HugePages: 0 pages'
  - 'Module nvme_tcp is not loaded: failed to execute: nsenter [--mount=/host/proc/204896/ns/mnt --net=/host/proc/204896/ns/net grep nvme_tcp /proc/modules], output , stderr : exit status 1'
  - 'Module uio_pci_generic is not loaded: failed to execute: nsenter [--mount=/host/proc/204896/ns/mnt --net=/host/proc/204896/ns/net grep uio_pci_generic /proc/modules], output , stderr : exit status 1'
  info:
  - Service iscsid is running
  - NFS4 is supported
  - Package nfs-common is installed
  - Package open-iscsi is installed
  - CPU instruction set sse4_2 is supported
  warn:
  - multipathd.service is running. Please refer to https://longhorn.io/kb/troubleshooting-volume-with-multipath/ for more information.
```

> Use the `install` sub-command to perform preflight dependencies installation and setup before installing Longhorn.

Reviewer: ```suggestion
Use the `install` sub-command to install and set up the preflight dependencies before installing Longhorn.
```

Author: Modified. Thanks.

```shell
master:~# ./longhornctl install preflight --enable-spdk
INFO[2024-01-01T00:00:03Z] Initializing preflight installer
INFO[2024-01-01T00:00:03Z] Cleaning up preflight installer
INFO[2024-01-01T00:00:03Z] Running preflight installer
INFO[2024-01-01T00:00:03Z] Installing dependencies with package manager
INFO[2024-01-01T00:00:10Z] Installed dependencies with package manager
INFO[2024-01-01T00:00:10Z] Cleaning up preflight installer
INFO[2024-01-01T00:00:10Z] Completed preflight installer. Use 'longhornctl check preflight' to check the result.
```

> After executing the `install` sub-command, you can use the `check` sub-command again to ensure all environment settings are correct:

Reviewer: ```suggestion
After installing and setting up the preflight dependencies, you can run the `check` sub-command again to verify that all environment settings are correct.
```

Author: Modified. Thanks.

```shell
master:~# ./longhornctl check preflight --enable-spdk
INFO[2024-01-01T00:00:13Z] Initializing preflight checker
INFO[2024-01-01T00:00:13Z] Cleaning up preflight checker
INFO[2024-01-01T00:00:13Z] Running preflight checker
INFO[2024-01-01T00:00:16Z] Retrieved preflight checker result:
worker1:
  info:
  - Service iscsid is running
  - NFS4 is supported
  - Package nfs-common is installed
  - Package open-iscsi is installed
  - CPU instruction set sse4_2 is supported
  - HugePages is enabled
  - Module nvme_tcp is loaded
  - Module uio_pci_generic is loaded
```

> #### With The Script

Reviewer: ```suggestion
#### Using the Script
```

Author: Modified. Thanks.
